filebeat/module/kibana/log/ingest/pipeline-ecs.yml

```diff
@@ -1,16 +1,36 @@
-description: Pipeline for parsing Kibana ecs logs
+description: Pipeline for parsing Kibana ECS logs
 processors:
 - set:
     field: event.ingested
     value: '{{_ingest.timestamp}}'
 - set:
     copy_from: '@timestamp'
     field: event.created
-- script:
-    lang: painless
-    inline: 'ctx.json.keySet().each (key -> ctx[key] = ctx.json.get(key))'
-- remove:
-    field: json
+- rename:
```

Comment:
Consistent with integration packages. These lines are responsible for making this fix to work:

```
add_to_root: true
add_to_root_conflict_strategy: merge
```